jax_unirep/evotuning_models.py

```diff
@@ -57,6 +64,7 @@ def mlstm256():
 def mlstm64():
     """mLSTM64 model functions."""
```

Comment:
```suggestion
    """Return mLSTM256 model's initialization and forward pass functions.
    
    The initializer function returned will give us random weights as a starting point.
    
    The model forward pass function will accept any weights compatible with those
    generated by the initializer function.
    The model implemented here has a trainable embedding,
    four consecutive mLSTM layers each with 64 nodes,
    and a single dense layer to predict the next amino acid identity.

    This is the simplest model published by the original UniRep authors.
    """
```

Reply:
There's still mLSTM256 mentioned in the first line. Not sure if you can easily correct that, then I can commit.

Reply:
Fixed!